src/components/ImageView/index.tsx

```diff
@@ -121,10 +101,8 @@ function ImageView({isAuthTokenRequired, url, fileName, onError}) {
         return {offsetX, offsetY};
     };
 
-    /**
-     * @param {SyntheticEvent} e
-     */
-    const onContainerPress = (e) => {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const onContainerPress = (e: any) => {
```

Comment:
The native event here is PointerEvent. Should we just convert the type for this case? I think we should bring this to slack